kitchen/runtimes/super-runtime/src/lib.rs

```diff
@@ -360,6 +379,10 @@ construct_runtime!(
 		BasicToken: basic_token::{Module, Call, Storage, Event<T>},
 		CheckMembership: check_membership::{Module, Call, Storage, Event<T>},
 		ScheduleOnFinalize: schedule_on_finalize::{Module, Call, Storage, Event<T>},
+		LastCaller1: last_caller::<Instance1>::{Module, Call, Storage, Event<T>},
+		LastCaller2: last_caller::<Instance2>::{Module, Call, Storage, Event<T>},
+		DefaultInstance: default_instance::{Module, Call, Storage, Event<T>},
```

Comment:
```suggestion
		DefaultInstance1: default_instance::{Module, Call, Storage, Event<T>},
```
You previously named this `DefaultInstance1` and I committed over it during the fix. Feel free to change it back, not sure how much it matters...